src/vario/buttons.h

```diff
@@ -1,37 +1,50 @@
-#ifndef buttons_h
-#define buttons_h
+#pragma once
 
 #include <Arduino.h>
 
-
-//Pinout for Leaf V3.2.0
-#define BUTTON_PIN_CENTER  2  // INPUT
-#define BUTTON_PIN_LEFT    3  // INPUT
-#define BUTTON_PIN_DOWN    4  // INPUT
-#define BUTTON_PIN_UP      5  // INPUT
-#define BUTTON_PIN_RIGHT   6 // INPUT
+// Pinout for Leaf V3.2.0
+#define BUTTON_PIN_CENTER 2 // INPUT
+#define BUTTON_PIN_LEFT 3   // INPUT
+#define BUTTON_PIN_DOWN 4   // INPUT
+#define BUTTON_PIN_UP 5     // INPUT
+#define BUTTON_PIN_RIGHT 6  // INPUT
 
 /*
 //Pinout for Breadboard
-#define BUTTON_PIN_UP     40  
+#define BUTTON_PIN_UP     40
 #define BUTTON_PIN_DOWN   38
 #define BUTTON_PIN_LEFT   39
 #define BUTTON_PIN_RIGHT  42
 #define BUTTON_PIN_CENTER 41
 */
 
-enum buttons {NONE, UP, DOWN, LEFT, RIGHT, CENTER, BOUNCE};
-enum button_states {NO_STATE, PRESSED, RELEASED, HELD, HELD_LONG};
-
-uint8_t buttons_init(void);
-
-uint8_t buttons_check(void);
-uint8_t buttons_inspectPins(void);
-uint8_t buttons_debounce(uint8_t button);
-uint8_t buttons_get_state(void);
+// D pad button states.
+// NOTE:  Left is -1 as to make casting to an int for settings easier
```

Comment:
This worries me just a little since it seems like semantically UP and DOWN might make sense as +1 and -1 at least as much or more than LEFT/RIGHT -- I'd recommend we don't try to encode numeric information into the enum so as to avoid the surprise that LEFT/RIGHT has valid numeric interpretation but DOWN/UP doesn't

Reply:
Seems like this note can be removed since we don't have numeric values here any more?